docs/api-gateway/docs/011-api-custom-resource.md

```diff
@@ -35,8 +35,6 @@ This table analyses the elements of the sample CR and the information it contain
 
 | Field   |      Mandatory?      |  Description |
 |:----------:|:-------------:|:------|
-| **apiVersion** |  **YES** | Defined basing on the **group** and **version** fields of the CRD **spec** section. |
```

Comment:
Why delete these fields? They are a part of every CR.

Reply:
Yup, that's why @derberg decided not to include them in the table as they are obvious.

Reply:
exactly, these are basic fields that are part of every resource, not only CRDs

Reply:
okaaaay